olad/RDMHTTPModule.cpp

```diff
@@ -3438,7 +3438,7 @@ void RDMHTTPModule::SendCurveResponse(HTTPResponse *response,
 
   for (unsigned int i = 1; i <= info->total; i++) {
     if (i <= info->curve_descriptions.size() &&
-        info->curve_descriptions[i - 1].length() > 0) {
+        info->curve_descriptions[i - 1].empty() == false) {
```

Comment:
You can just use:
`!info->curve_descriptions[i - 1].empty()`

So the total bit is:
```
    if (i <= info->curve_descriptions.size() &&
        !info->curve_descriptions[i - 1].empty()) {
```

Which should be more readable.

Reply:
Sure thing, happy to change that. I've found using `== false` makes it more explicit in my personal coding.